.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set env
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
```

Comment:
I think we can simply push it to GitHub registry

Reply:
I wouldn't excluding pushing to GitHub registry, but there is precedence for using DockerHub for OTel images.  It seems the community is using both locations (operator only used GitHub, demo and collector publish to both).  As long as I have an image to use I'll be happy so I don't care where it lives.  If starting with only Github is easier then that works for me.

Reply:
I just have no idea if the secrets are available.

Reply:
https://github.com/open-telemetry/community/issues/1377 is completed so the secrets in `DOCKER_USERNAME` and `DOCKER_PASSWORD` should be available to us if we chose to use them.

Reply:
Updated to push to both.